test/assignment3/Assignment3Test.java

```diff
@@ -0,0 +1,81 @@
+package assignment3;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class Assignment3Test {

```

Comment:
Tests for some other cases, like empty grid, grid with no words from the dictionary?